### DISCUSSION TITLE
remove `chakra-*` class names from prod builds

### DISCUSSION THREAD

**Discussion by MariuzM** [chakra-ui/chakra-ui#2683] [Q&A]
Is there an option to remove them in prod builds. I personally don't need them. They are just making html look more cluttered, they are not being in used and if it will be i will define a class myself.
![Screen Shot 2020-12-01 at 20 56 51](https://user-images.githubusercontent.com/36937039/100795966-d3f6ef80-3417-11eb-80c0-11723254fec0.png)

**Comment by sengho66** [comment]
Is there a solution for this?

**Comment by ahmedelq** [comment]
Unfortunately, it seems to be hard-coded.
